nagios.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+// NagiosStatusData is a task factory for tasks that fetch a JSON representation of
+// the provided project. The task uses outFor and errOutFor to get io.Writers
+// to write, respectively, the JSON output and any eventual error message.
+func GetNagiosStatusData(project, pod string, outFor, errOutFor projectResourceWriterCloserFactory) Task {
+	return retrieveNagiosData(func() *exec.Cmd {
+		return exec.Command("oc", "exec", pod, "--", "cat", "/var/log/nagios/status.dat")
+	}, project, pod, "status", outFor, errOutFor)
+}
+
+// NagiosStatusData is a task factory for tasks that fetch a JSON representation of
+// the provided project. The task uses outFor and errOutFor to get io.Writers
+// to write, respectively, the JSON output and any eventual error message.
+func GetNagiosHistoricalData(project, pod string, outFor, errOutFor projectResourceWriterCloserFactory) Task {
+	return retrieveNagiosData(func() *exec.Cmd {
+		return exec.Command("oc", "exec", pod, "--", "tar", "-c", "-C", "/var/log/nagios", "archives", "--to-stdout")
```

Comment:
According to `man tar`:

```
    -O, --to-stdout
              Extract files to standard output.
```

It doesn't seen to have any relation to creating archives?

In a quick local test `--to-stdout` seems to have no influence on `tar c foo > foo.tar`.